### ISSUE TITLE
Remove gray background from Qualtrics survey, except on mobile if possible

### ISSUE THREAD

**Issue by evwilkin** [patternfly/patternfly-org#1743]
The gray background/overlay prevents users from interacting with the site and should be removed except on mobile where the survey does block most of the page

**Comment by ddonahue007** [comment]
Including snippet from slack

```
Doug Donahue  6 days ago
I did a bit more investigation and it appears that the greyed out background and the survey taking focus, is due to the type of survey(PopOver) that we are using.
Pop Overs are active Creatives, meaning visitors must interact with your Pop Over before they can proceed to the rest of your website. Because of the way a Pop Over interrupts visitors, visitors are more likely to follow-through with the desired action (for example, if you want them to complete a feedback survey).

Doug Donahue  6 days ago
There seems to be other creative options if we prefer to move away from the Popover. https://www.qualtrics.com/support/website-app-feedback/creatives-tab/creative-types/pop-over-creative/

Doug Donahue  6 days ago
I am including @sarachizari in the conversation as she was involved in the creation of the survey itself.```

**Comment by ddonahue007** [comment]
Additional  slack snippet

```
sarachizari  13 hours ago
sorry just saw this conversation. popover is the most common creative in websites feedback world, but feedback button is also a good option to use. please let me know if you decide to keep the survey and want to change the type of creative @Rachael Petrie @Doug Donahue

sarachizari  12 hours ago
and if PatternFly website stores cookies, we can use that to only target return visitors for the survey..I’d need the name of the cookie we have for return visitors on the website
```

**Comment by evwilkin** [comment]
Closed by #1760
